cves/kernel/CVE-2013-1796.yml

```diff
@@ -1,259 +1,117 @@
-  as finished unless curated is properly updated.
-curation_level: 0
-reported_instructions: |
-  What date was the vulnerability reported to the security team? Look at the
-  security bulletins and bug reports. It is not necessarily the same day that
-  the CVE was created.  Leave blank if no date is given.
-
-  Please enter your date in YYYY-MM-DD format.
-reported_date:
-announced_instructions: |
-  Was there a date that this vulnerability was announced to the world? You can
-  find this in changelogs, blogs, bug reports, or perhaps the CVE date.
-
-  This is not the same as published date in the NVD - that is below.
-
-  Please enter your date in YYYY-MM-DD format.
-announced_date: '2013-03-22'
-published_instructions: |
-  Is there a published fix or patch date for this vulnerability?
-  Please enter your date in YYYY-MM-DD format.
-published_date: '2013-03-22'
-description_instructions: |
-  You can get an initial description from the CVE entry on cve.mitre.org. These
-  descriptions are a fine start, but they can be kind of jargony.
-
-  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
-  read to anyone with some programming experience. We can always pull up the NVD
-  description later to get more technical.
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
-bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
-bounty:
-  amt:
-  announced:
-  url:
-reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
-
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
+CWE: [119]
+ipc:
+  question: |
+    Did the feature that this vulnerability affected use inter-process
+    communication? IPC includes OS signals, pipes, stdin/stdout, message
+    passing, and clipboard. Writing to files that another program in this
+    software system reads is another form of IPC.
+    
+    Answer must be true or false.
+    Write a note about how you came to the conclusions you did, regardless of
+    what your answer was.
+  answer: false
+  note: |
+    The vulnerability deals with buffer overflows, which doesn't involve inter-process communication.
+CVSS: ::"6.8"
 bugs: []
-fixes_instructions: |
-  Please put the commit hash in "commit" below.
+i18n:
+  note: |
```

Comment:
This note should just be included in the note below about i18n. Not seperated into two differnet answers about i18n.